Compiler/src/typeinfer.jl

```diff
@@ -1292,7 +1294,7 @@ function typeinf_ext_toplevel(methods::Vector{Any}, worlds::Vector{UInt}, trim::
             mi = get_ci_mi(callee)
             def = mi.def
             if use_const_api(callee)
-                src = codeinfo_for_const(interp, mi, code.rettype_const)
+                src = codeinfo_for_const(interp, mi, callee.rettype_const)
```

Comment:
Were we not hitting this code path before?

That's... kind of frightening

Reply:
Yep looks like it!